docs/application/dotnet/guides/alarm/alarms.md

```diff
@@ -56,6 +56,9 @@ To enable your application to use the alarm functionality:
 
 You can set an alarm which, when it expires, either launches an application or sends a notification to the user:
 
+> [!NOTE]
+> Since Tizen 6.0, the time period value of an alarm can be one of the values of `Tizen.Applications.AlarmStandardPeriod`. This time period value will be phase-aligned with another time period value of the alarm. For the `CreateAlarm(int, Tizen.Applications.AlarmStandardPeriod, Tizen.Applications.AppControl)` or `CreateAlarm(int, Tizen.Applications.AlarmStandardPeriod, Tizen.Applications.Notifications.Notification)` method of the `Tizen.Applications.AlarmManager` class. If you use the `Tizen.Applications.AlarmStandardPeriod`, the time period of the alarm is guaranteed.
```

Comment:
"Since Tizen 6.0, the time period value of an alarm can be one of the values of `Tizen.Applications.AlarmStandardPeriod`. "

=> @Reni-90 How about using simple sentence? For example,
Since Tizen 6.0, `Tizen.Applications.AlarmStandardPeriod` controls(or manages) time period value of an alarm.

"This time period value will be phase-aligned with another time period value of the alarm. "

=> @kilig To me also, this sentence is not clear. What this sentence mean?

"For the `CreateAlarm(int, Tizen.Applications.AlarmStandardPeriod, Tizen.Applications.AppControl)` or `CreateAlarm(int, Tizen.Applications.AlarmStandardPeriod, Tizen.Applications.Notifications.Notification)` method of the `Tizen.Applications.AlarmManager` class. "

=> This sentence seems to be not in complete form. (It is in _For the A or B._ form) Would you please check?
What did you intend to mean with this?

Reply:
@mijin85cho 
I agree with your suggestion. In a note, it is better to give a simple and complete sentence. However, will wait for @kilig's response.

What I understand from the second sentence is:
For the `CreateAlarm (int, Tizen.Applications.AlarmStandardPeriod, Tizen.Applications.AppControl)` or `CreateAlarm (int, Tizen.Applications.AlarmStandardPeriod, Tizen.Applications.Notifications.Notification)` method of the `Tizen.Applications.AlarmManager` class, if you use the `Tizen.Applications.AlarmStandardPeriod`, the time period value of the alarm is guaranteed.

@kilig 
Please verify if the technical meaning is intact.

Reply:
I intended that 'Tizen.Applications.AlarmStandardPeriod' is for the A or B.

Reply:
So, are we trying to say:

For the `CreateAlarm` method with `Tizen.Applications.AppControl` or `Tizen.Applications.Notifications.Notification`, if you use the `Tizen.Applications.AlarmStandardPeriod`, the time period value of the alarm is guaranteed.

Please verify and correct me if I am wrong.